Disher/View Controllers/ListContentViewController.m

```diff
@@ -99,7 +100,7 @@ - (void)tableView:(UITableView *)tableView commitEditingStyle:(UITableViewCellEd
         }];
     }
 }
-- (void) reloadListRecipes:(id)something completionHandler:(void(^)(NSArray *returnedRecipes))completionHandler{
+- (void)reloadListRecipes:(id)something completionHandler:(void(^)(NSArray *returnedRecipes))completionHandler{
```

Comment:
just noticing this `something`. can we rename

Reply:
solved; didnt know easier syntax when I wrote this